### PR TITLE
Add new social medias to footer

### LINKED ISSUE #1062: Add More Socials to Footer
With the new home page, we need to update our footer to feature our most used socials (Discord and Insta).

### PR DESCRIPTION
**Pull request by neelp03**
## Resolves #1062 
- added new socials to the footer
- changed styling to match the new design

### Before: 
image

### After:
image

### CODE REVIEW

**Review by andrewtedi — APPROVED** [review]
Looks amazing!

**Review by evanugarte — COMMENTED** [review]
consider making these icons their own compoents in [`src/Components/Footer`](https://github.com/SCE-Development/Core-v4/tree/dev/src/Components/Footer) like:

```jsx
import React from 'react';

function DiscordIcon() {
  return (
    <React.Fragment>
      {/* your svg stuff here */}
    </React.Fragment>
  )
}

```

do this for each icon, then the changes in the footer becomes tiny

File: src/Components/Footer/footer.css [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+
+.footer-text a svg {
+  margin: 5px;
+}
+
+@media (max-width: 500px) {
+  #social-media {
+    margin-left: 50px;
+  }
+}
```
Comment (evanugarte) [inline src/Components/Footer/footer.css:38]:
```suggestion
}

```

File: src/Components/Footer/Footer.js [hunk 2]
```diff
@@ -1,63 +1,55 @@
...
-              />
-            </svg>
-          </a>
-
+        <Col id='social-media'>
+          <a
+            href='https://www.linkedin.com/company/sjsu-software
+          -computer-engineering-society/'>{<LinkedinIcon />}</a>
+          <a href='https://www.facebook.com/sjsusce/'>{<FacebookIcon />}</a>
+          <a href='https://www.instagram.com/sjsusce/'>{<InstagramIcon />}</a>
```
Comment (evanugarte) [inline src/Components/Footer/Footer.js:51]: can the href live in the component? if not that's ok

**Review by evanugarte — APPROVED** [review]
if it works it works

**Pull request merged**